app/forms/provider_interface/offer_wizard.rb

```diff
@@ -3,32 +3,46 @@ class OfferWizard
     include Wizard
     include Wizard::PathHistory
 
-    SKE_LENGTH = 8.step(by: 4).take(6).map do |length|
-      "#{length} weeks"
-    end.freeze
+    SKE_LENGTH = 8.step(by: 4).take(6).freeze
+    SKE_LANGUAGES = [
+      'French',
+      'Spanish',
+      'German',
+      'ancient languages',
+    ].freeze
+    SKE_LANGUAGES_WITH_NO = [SKE_LANGUAGES, 'no'].flatten.freeze
```

Comment:
it's not immediately obvious what `SKE_LANGUAGES_WITH_NO` is. not often i suggest this, but perhaps a comment is in order.